.copier-answers.yml

```diff
@@ -1,15 +1,24 @@
 # Do NOT update manually; changes here will be overwritten by Copier
-_commit: v1.5.1-6-gd28f504
+_commit: v2.1.4
 _src_path: gh:coopiteasy/oca-addons-repo-template
```

Comment:
This is incorrect. The OCA template needs to be used. You probably need to do something like `copier copy https://github.com/OCA/oca-addons-repo-template.git .` to override all changes.